packages/styles/scss/themes/expressive/components/_tag.scss

```diff
@@ -16,6 +16,71 @@
 /// @group tag-expressive
 @mixin tag-expressive {
   .#{$prefix}--tag {
+    @include carbon--type-style('body-long-01');
+
     height: carbon--rem(32px); /* carbon--container-02 */
+    padding: 6px $spacing-05;
+  }
+
+  .#{$prefix}--tag--blue {
+    color: $carbon--blue-80;
+  }
+
+  .#{$prefix}--tag--red {
+    color: $carbon--red-80;
+  }
+
+  .#{$prefix}--tag--magenta {
+    color: $carbon--magenta-80;
+  }
+
+  .#{$prefix}--tag--purple {
+    color: $carbon--purple-80;
+  }
+
+  .#{$prefix}--tag--cyan {
+    color: $carbon--cyan-80;
+  }
+
+  .#{$prefix}--tag--teal {
+    color: $carbon--teal-80;
+  }
+
+  .#{$prefix}--tag--green {
+    color: $carbon--green-80;
+  }
+
+  .#{$prefix}--tag--gray {
+    color: $carbon--gray-80;
+  }
+
+  .#{$prefix}--tag--cool-gray {
+    color: $carbon--cool-gray-80;
+  }
+
+  .#{$prefix}--tag--warm-gray {
+    color: $carbon--warm-gray-80;
+  }
+
+  .#{$prefix}--tag--filter {
+    padding-right: $spacing-04;
+  }
+
+  .#{$prefix}--tag--disabled,
+  .#{$prefix}--tag--filter.#{$prefix}--tag--disabled {
+    color: $disabled-02;
+  }
```

Comment:
Wondering the color styles in here were actually needed, as they are [implemented in Carbon core](https://github.com/carbon-design-system/carbon/blob/v10.21.0/packages/components/src/components/tag/_tag.scss#L48-L139).

Reply:
The main difference between these and those is that the colors here are 80 grade while the others are 70 grade. One of the new things in the Expressive theme is the darkening of the colors. But if there's another way to get 80 grade colors from the Carbon core classes that I'm missing, please let me know!

Reply:
> One of the new things in the Expressive theme is the darkening of the colors.

Thank you for pointing it out @IgnacioBecerra! Sounds that we need more system-wide change, similar to what we do for type tokens. I think we can start with changing color tokens at https://github.com/carbon-design-system/ibm-dotcom-library/blob/v1.11.0/packages/styles/scss/themes/expressive/_tokens.scss#L371-L479.

@jeffchew Is this something worthwhile as a engineering huddle topic?

Reply:
Sorry, I missed this comment. Let's bring up as a huddle topic @IgnacioBecerra @asudoh .

Reply:
Cool, done!